src/openvpn/options.c

```diff
@@ -851,7 +851,11 @@ init_options(struct options *o, const bool init_gc)
     o->scheduled_exit_interval = 5;
 #endif
 #ifdef ENABLE_CRYPTO
+#ifdef ENABLE_CRYPTO_WOLFSSL
+    o->ciphername = "AES-256-CBC";
+#else
     o->ciphername = "BF-CBC";
+#endif
```

Comment:
Hm this is still not okay.

Reply:
This is removed in the next commit. Please take a look [here](https://github.com/OpenVPN/openvpn/pull/134/commits/765aa66ff7bf8d9a93bb5ee590d7292265928c79)